book/src/pickers.md

```diff
@@ -0,0 +1,11 @@
+## Using pickers
+
+Helix has a variety of pickers, which are interactive windows used to select various kinds of items. These include a file picker, global search picker, and more. Most pickers are accessed via keybindings in [space mode](./keymap.md#space-mode). Pickers have their own [keymap](./keymap.md#picker) for navigation.
+
+### Filtering Picker Results
+
+Most pickers perform fuzzy matching using [fzf syntax](https://github.com/junegunn/fzf?tab=readme-ov-file#search-syntax). Two exceptions are the global search picker, which uses regex, and the workspace symbol picker, which passes search terms to the LSP.
```

Comment:
Is it worth reproducing the fzf syntax table in the helix docs?

Reply:
Let's link out to fzf docs for now. If we start to diverge or want a more stable page of docs for it we can add it later, either here or maybe in [nucleo](https://github.com/helix-editor/nucleo)'s README